tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -314,10 +314,12 @@ port::Status CudnnSupport::Init() {
     } else {
       const auto& version = result.ValueOrDie();
       LOG(INFO) << "running driver version: " << DriverVersionToString(version);
+#if !defined(__APPLE__)
```

Comment:
Agreed.  I haven't been able to find a public API for getting the kernel driver version that is reported by OSX CUDA preferences panel :(